x-pack/plugins/cloud_integrations/cloud_data_migration/public/plugin.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreSetup, CoreStart, Plugin } from '@kbn/core/public';
+import { ManagementAppMountParams } from '@kbn/management-plugin/public';
+import { CloudDataMigrationPluginSetup, CloudDataMigrationPluginStart } from './types';
+import { PLUGIN_ID, PLUGIN_NAME } from '../common';
+import { BreadcrumbService } from './application/services/breadcrumbs';
+
+export class CloudDataMigrationPlugin
+  implements Plugin<CloudDataMigrationPluginSetup, CloudDataMigrationPluginStart>
+{
+  private breadcrumbService = new BreadcrumbService();
+
+  public setup(core: CoreSetup, { management }: CloudDataMigrationPluginSetup) {
```

Comment:
I think we'll also need some conditional logic here so that the page isn't accessible when on cloud